README.md

```diff
@@ -39,19 +39,25 @@ mounted, persistent volumes.  More on this in [the ostree docs](https://ostreede
 
 ## Status
 
-At the current time, bootc is in active development and is not quite
-considered ready for production use.  The command line interface
-*might* change.  There is not yet stable RPC API.
+The core `bootc update` functionality is really just the same
+technology which has shipped for some time in (rpm-ostree) so there
+should be absolutely no worries about using it for OS updates.
+If you're using `rpm-ostree` today in container mode, you should
+have absolutely no worries about installing and using
+`bootc update` instead.  (More on this below)
```

Comment:
This paragraph is a bit repetitive. Did you mean to have just one of the two sentences there?

Reply:
Yes thanks!